File: Python/Task255.py
import math #для обчислення кореня квадратного
print('Задайте відрізок для змінної x')
a = float(input())
b = float(input())
x = a; hx = (b - a) / 7
print('Задайте відрізок для змінної y')
c = float(input())
d = float(input())
y = c; hy = (d - c) / 7
print(" y\\x ", end ="")
for i in range(0, 8):
    print("%8.2f" % (x + i*hx), end ="")
print() # перехід виводу на нову стрічку
#Зовнішній цикл за аргументом y
for i in range(0,8):
    print("%5.2f" % y, end ="")
    x = a
    #вкладений цикл за аргументом х
    for j in range(0,8):
        u = math.sqrt(2 * x * y + 4 * y * y * y)
        print("%8.2f" % u, end ="")
        x += hx
    print()
    y += hy

File: Kotlin/Task255.kt
import kotlin.math.sqrt

fun main() {
    println("Введіть межі для x")
    val a: Double = readln().toDouble()
    val b: Double = readln().toDouble()
    var x = a
    val hx = (b - a) / 7
    println("Введіть межі для y")
    val c: Double = readln().toDouble()
    val d: Double = readln().toDouble()
    var y = c
    val hy = (d - c) / 7
    var u: Double
    print("y\\x  ")
    for (i in 0..7) {
        print("%8.2f".format(x + hx * i))
    }
    println()
    for (i in 0..7) {
        print("%5.2f".format(y))
        x = a
        for (j in 0..7) {
            u = sqrt(2 * x * y + 4 * y * y * y)
            print("%8.2f".format(u))
            x += hx
        }
        println()
        y += hy
    }
}
